src/traces/pie/plot.js

```diff
@@ -761,7 +764,7 @@ function positionTitleInside(cd0) {
     return {
         x: cd0.cx,
         y: cd0.cy,
-        scale: cd0.trace.hole * cd0.r * 2 / textDiameter,
+        scale: (cd0.trace.hole[0] || cd0.trace.hole) * cd0.r * 2 / textDiameter,
```

Comment:
Also here we should use `Lib.isArrayOrTypedArray` to test `trace.hole` before try accessing its index.

Reply:
This block is for an inside title - so to be safe I think it'll need to use the minimum value in the `hole` array, rather than the first one, when it's an array.

Reply:
Done :) Now I do the array check and grab the min value if it's an array